qemu/tests/virtual_nic_private.py

```diff
@@ -28,6 +31,27 @@ def data_mon(session, cmd, timeout):
                 test.fail("God! Capture the transfet data:'%s'" % str(e))
             logging.info("Guest3 catch data is '%s'" % str(e))
 
+    def windump_env_set(vm, session):
+        """
+        Setup windump env in windows guest
+
+        :param vm: guest to install pcap and set up windump
+        :param session: guest session
+        """
+        error_context.context("Copy windump and npcap to "
+                              "windows guest", logging.info)
+        host_windump_dir = os.path.join(data_dir.get_deps_dir(),
```

Comment:
The deps_dir is mainly for other python scripts, not very suitable for windows tools. Maybe you can to try put the windump.exe to winutils.iso.

Reply:
@kimi1978wy @pezhang Updated. Please help review again. Thanks.